packages/vite/src/node/server/pluginContainer.ts

```diff
@@ -454,15 +454,19 @@ export async function createPluginContainer(
         err.frame = err.frame || generateCodeFrame(err.id!, err.loc)
       }
 
-      if (err.loc && ctx instanceof TransformContext) {
+      if (
+        ctx instanceof TransformContext &&
+        typeof err.loc?.line === 'number' &&
+        typeof err.loc?.column === 'number'
+      ) {
         const rawSourceMap = ctx._getCombinedSourcemap()
         if (rawSourceMap) {
           const traced = new TraceMap(rawSourceMap as any)
           const { source, line, column } = originalPositionFor(traced, {
             line: Number(err.loc.line),
             column: Number(err.loc.column),
           })
-          if (source && line != null && column != null) {
+          if (source) {
```

Comment:
Shouldn't we still guard here against `originalPositionFor` returning a [`InvalidOriginalMapping`](https://github.com/jridgewell/trace-mapping/blob/23ab4a29244444d825044ff8fb9a8d57f41837fa/src/types.ts#L42)
```suggestion
          if (source && line != null && column != null) {
```

Reply:
Yeah, I accidentally misidentified l with `line` and `err.loc.line`. 😅 Corrected.